__tests__/getOptions.spec.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-undef */
+const { options,
+    getAppOptions,
+    getEnvOptions } = require('./../utils/getOptions');
+const constants = require('./../constants/index');
+const fs = require('fs');
+const process = require('process');
+
+describe('Get Options script', () => {
+    const OLD_ENV = process.env;
+    const processCwdValue = process.cwd();
+
+    beforeEach(() => {
+        jest.resetModules();
+        process.env = { ...OLD_ENV };
+        delete process.env.NODE_ENV;
+    });
+
+    afterEach(() => {
+        jest.clearAllMocks();
+        process.env = OLD_ENV;
+    });
+
+    test('getEnvOptions: should return empty envOptions object if environments don\'t set', () => {
```

Comment:
What do you think of splitting different functions into different nested describes?
```
getEnvOptions
|-getEnvOptions
  |-test
  |-test
  |-...
|-getAppOptions
  |-...
|-options
  |-...
```

Reply:
This is a cool comment, I will do it.